spearheadblocks/style.css

```diff
@@ -0,0 +1,32 @@
+/*
+Theme Name: Spearhead Blocks
+Theme URI: https://github.com/automattic/themes/tree/trunk/spearhead-blocks
+Author: Automattic
+Author URI: https://automattic.com
+Description: Spearhead Blocks is the Full Site Editing capable, block based version of the original Spearhead theme.
+Requires at least: 5.8
+Tested up to: 5.9
+Requires PHP: 5.7
+Version: 0.0.1
+License: GNU General Public License v2 or later
+License URI: https://raw.githubusercontent.com/Automattic/themes/trunk/LICENSE
+Template: blockbase
```

Comment:
Consider using `blank-canvas` as a parent instead of Blockbase.  I gave it a quick test and it seems to have the same design.  Blank Canvas has less "baggage" than Blockbase.  

We haven't shipped Blank Canvas as a theme yet (thus it couldn't be used as a parent in production) but if there are viable usecases for it (such as this theme) it would be worth considering.